src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -74,10 +74,10 @@ private void testCreateInstructorAction() throws Exception {
         InstructorAttributes instructor = new InstructorAttributes();
         
         String shortName = "Instrúctör";
-        instructor.name = "AHPUiT Instrúctör";
-        instructor.email = "AHPUiT.instr1!@gmail.tmt";
+        instructor.name = "AHPUiT Instrúctör+++";
```

Comment:
Can this made more realistic? e.g. `AHPUiT Instrúctör WithPlusInEmail` Names don't have `+` in them. Unrealistic data makes the test outputs look unrealistic, making it hard to detect rubbish output caused by bugs.